app/src/main/java/com/x8bit/bitwarden/data/platform/manager/di/PlatformManagerModule.kt

```diff
@@ -80,12 +80,12 @@ object PlatformManagerModule {
 
     @Provides
     @Singleton
-    fun provideBridgeServiceProcessor(
-        bridgeRepository: BridgeRepository,
+    fun provideAuthenticatorBridgeServiceProcessor(
```

Comment:
While we're changing the name here I think we can remove the word "Service" from here since it doesn't match the actual class name.

Reply:
👍 Nice, I didn't even notice that.